pkgs/development/python-modules/cyclonedds-python/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+, cyclonedds
+, setuptools
+, rich-click
+
+, pytestCheckHook
+, pytest-mock
+}:
+
+buildPythonPackage rec {
+  pname = "cyclonedds-python";
+  version = "0.10.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "eclipse-cyclonedds";
+    repo = "cyclonedds-python";
+    rev = "refs/tags/${version}";
+    hash = "sha256-MN3Z5gqsD+cr5Awmsia9+uCHL/a2KQP2uMS13rVc1Hw=";
+  };
+
+  postPatch = ''
+    sed -i 's/^addopts.*/addopts = "-ra --import-mode=importlib"/' pyproject.toml
+    sed -i '/^required_plugins/ s/pytest-cov//' pyproject.toml
+  '';
```

Comment:
```suggestion
```

We now have `pytest-cov-stub` instead, which you can add to `nativeCheckInputs`.